File: 13-01-2025_Minimum_Length_of_String_After_Operations.py
class Solution:
    def minimumLength(self, s: str) -> int:
        freq_map = Counter(s)
        min_len = 0
        for count in freq_map.values():
            min_len += 2 if count % 2 == 0 else 1
        
        return min_len

File: 13-01-2025_Minimum_Length_of_String_After_Operations.java
class Solution {
    public int minimumLength(String s) {
        HashMap<Character, Integer> freqMap = new HashMap<>();
        for (char c : s.toCharArray()) {
            freqMap.put(c, freqMap.getOrDefault(c, 0) + 1);
        }
        
        int minLen = 0;
        for (int count : freqMap.values()) {
            minLen += (count % 2 == 0) ? 2 : 1;
        }
        
        return minLen;
    }
}
